src/components/Calculator.js

```diff
@@ -1,38 +1,52 @@
-import React from 'react';
+import React, { useState } from 'react';
+import calculate from '../logic/calculate';
 
 export default function CalculatorComp() {
+  const [currentState, setState] = useState({ total: 0, next: null, operation: null });
+
+  const handleClick = (e) => {
+    setState((currentState) => calculate(currentState, e.target.textContent));
+    if (e.target.textContent === 'AC') {
+      setState({ total: 0, next: null, operation: null });
+    }
+  };
```

Comment:
- [ ] I can not find where we passed `props` from the parent component to the child component in our application as requested from the previous code reviewer. For more information about `props`, kindly visit this [link](https://react.dev/learn/passing-props-to-a-component)

Reply:
@addod19 Hello! I'll really appreciate if you can clear some doubts about this project. 
I think I've built a child component already. This child component is created in another file, named CalculatorComponent.js
That's why you are not going to find the child in this Calculator.js, Can you check the CalculatorComponent.js, please? So you can tell me if this is okay.

Reply:
@addod19 Here are some screenshots of what I'm talking about
![event-react-1](https://user-images.githubusercontent.com/105683031/234653916-44cb10e9-dc9a-4fbc-8d01-57d30f7279c1.png)
![react-event-2](https://user-images.githubusercontent.com/105683031/234653966-b3ef06ac-cb3e-43c5-bf9e-29a7ed76b7fa.png)

Reply:
@addod19 Or I'm supposed to do it like this??

![react-event-3](https://user-images.githubusercontent.com/105683031/234654906-940f6212-d437-462f-a788-6ad13a81509a.png)
![react-event-4](https://user-images.githubusercontent.com/105683031/234654915-e8855784-01ff-4d4c-8609-ba5c3c89e113.png)